CMakeLists.txt

```diff
@@ -1,16 +1,22 @@
+# Setup CMake project
```

Comment:
This is standard CMake stuff which I don't think we need to explain in a comment.

Reply:
removed